src/Tribe/Admin/Helpers.php

```diff
@@ -190,4 +190,81 @@ public function is_action( $action = null ) {
 
 		return false;
 	}
+
+	/**
+	 * Get a list of the admin screens for showing notices, etc.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string> $additional A list of additional screens to add to the list.
+	 *
+	 * @return array<string> $screens The list of screen IDs.
+	 */
+	public static function get_admin_screens_list( $additional = [] ) {
+		// @TODO: some of these are redundant or old -this list should be audited.
+		$default_screens = array_merge(
+			self::get_settings_screens_list(),
+			self::get_app_shop_screens_list()
+		);
+
+		if ( empty( $additional ) ) {
+			return $default_screens;
```

Comment:
Bailing here means this value will never be filtered with `tec_admin_screens_list`. Maybe we can do a `wp_parse_args` without bailing and then returning the filtered value.